public/assets/scss/_base.scss

```diff
@@ -1,3 +1,26 @@
+// This file is part of SpongeHome, licensed under the MIT License (MIT).
+
```

Comment:
I read about scss block comments staying in the actual css file when using block comments.